appveyor.yml

```diff
@@ -19,29 +19,30 @@ environment:
     - TOOLCHAIN: "nmake-vs-12-2013"
       PROJECT_DIR: examples\benchmark
 
-    - TOOLCHAIN: "vs-10-2010"
-      PROJECT_DIR: examples\benchmark
+    # - TOOLCHAIN: "vs-10-2010"
+    #   PROJECT_DIR: examples\benchmark
 
-    - TOOLCHAIN: "vs-11-2012"
-      PROJECT_DIR: examples\benchmark
+    # - TOOLCHAIN: "vs-11-2012"
+    #   PROJECT_DIR: examples\benchmark
 
     - TOOLCHAIN: "vs-12-2013-win64"
       PROJECT_DIR: examples\benchmark
 
-    - TOOLCHAIN: "vs-12-2013-xp"
-      PROJECT_DIR: examples\benchmark
+    # - TOOLCHAIN: "vs-12-2013-xp"
+    #   PROJECT_DIR: examples\benchmark
 
     - TOOLCHAIN: "vs-12-2013"
       PROJECT_DIR: examples\benchmark
 
     - TOOLCHAIN: "vs-14-2015"
       PROJECT_DIR: examples\benchmark
 
-    - TOOLCHAIN: "vs-9-2008"
-      PROJECT_DIR: examples\benchmark
+    # - TOOLCHAIN: "vs-9-2008"
+    #   PROJECT_DIR: examples\benchmark
 
     - TOOLCHAIN: "mingw"
       PROJECT_DIR: examples\benchmark
+      CXX_PATH: 'C:\mingw-w64\i686-5.3.0-posix-dwarf-rt_v4-rev0\mingw32\bin'
```

Comment:
I have no idea why and how they are using it. Do we have install errors because of absent of this variable?